tabbycat/users/views.py

```diff
@@ -79,6 +79,16 @@ def get_form_kwargs(self):
     def get_success_url(self):
         return reverse_tournament('options-tournament-index', self.tournament)
 
+    def form_valid(self, form):
+        form.save(
+            domain_override=self.request.get_host(),
```

Comment:
Why do we need this override?

Also, we could include `extra_email_context=None` and avoid the `.get()` in the form.

```suggestion
            extra_email_context=None,
```

Reply:
Thanks, this works - and I've taken back out the .get() in the InviteUserForm.save method in forms.py. Just wanted to flag that might be worth updating this to something like 

```
kwargs['extra_email_context'] = {
    **(kwargs.get('extra_email_context') or {}),
    'tournament': self.tournament,
}
```

in future - this would be more consistent with Django’s own handling of optional keyword arguments in similar contexts and would avoid raising a KeyError if extra_email_context isn’t included in kwargs.